### DISCUSSION TITLE
trying the M1 version on M2

### DISCUSSION THREAD

**Discussion by kinoyoki** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#767] [General]
seems the one for M1 can be used on M2 models, no problems found at the moment, thanks for creating this wonderful app

**Comment by MatthiasKerbl** [comment]
> seems the one for M1 can be used on M2 models, no problems found at the moment, thanks for creating this wonderful app

Hi @Sequence63,

M2 is now officially supported as well.

Cheers,
Matthias
